### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by rosiel**
## Purpose / why

Fills in the stub Audio and Video pages, addressing #2139 
Fixes the broken link to Typed Relation, addressing #2141 

## What changes were made?

as listed above, plus
* added a mention that Islandora Defaults provides a View for collection display.

## Verification

* does it build
* do you know it to be inaccurate?

## Interested Parties

* @Islandora/documentation
* @Islandora/committers

### Pull-request Reviewer
Pull-request reviewer should ensure the following:

* [ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?

### Person Merging
The person merging should ensure the following:

* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

**Comment by ysuarez** [comment]
This looks pretty good. Need to look it over more closely.

File: docs/concepts/collection.md [hunk 1]
```diff
@@ -10,25 +10,44 @@ Collections are groups of related content that can be viewed or managed as a uni
...
+Bulk management of items can be done from a Views Bulk Operations-compatible View, including the main Drupal Content View (at admin/content) and the Manage Members View (at node/[node]/members). This is possible due to the Drupal contrib modules [Views Bulk Operations (VBO)](https://www.drupal.org/project/views_bulk_operations) and [Views Bulk Edit](https://www.drupal.org/project/views_bulk_edit). VBO allows you to perform Drupal Actions on objects. These Actions include making Islandora Derivatives such as "Audio - Generate a Service File from an Original File" and also to perform Drupal core Actions such as publishing/unpublishing content. The Views Bulk Edit module extends VBO and allows you to edit field values (such as the Member Of field, which would change which collection a group of nodes are in). 
+
+For instance, if you want to move a number of nodes from one collection into another, then you can
```
Comment (eldonquijote) [inline docs/concepts/collection.md:33]: The following list items don't show up as bullet points when I build the documentation page, I think they need to be indented in the source code

File: docs/concepts/collection.md [hunk 2]
```diff
@@ -10,25 +10,44 @@ Collections are groups of related content that can be viewed or managed as a uni
...
+* Select "Confirm" at the bottom of the form to save your changes.
 
 For more information see the video tutorial on [Batch Editing](https://youtu.be/ZMp0lPelOZw).
+
+
+## Deleting A Collection
+
+When you delete a Collection object, it will be deleted from the "Member of" field of all its Children. This means that it will not be possible to track down the children after the Collection has been deleted. If you intend to move the children, see above. If you intend to delete the children, this must be done before-hand. 
+
+CAREFUL! Deleting content (Collections or otherwise) will NOT delete any attached Media. This is a [known issue](https://github.com/Islandora/documentation/issues/909).
```
Comment (eldonquijote) [inline docs/concepts/collection.md]: Could we make this an admonition/callout?

File: docs/concepts/node-media.md [hunk 3]
```diff
@@ -62,8 +62,7 @@ This migration is performed by all installation methods, so these values should
 be available "out of the box". These values were provided by the PCDM data model
 (see [RDF in Islandora](rdf.md)),
 and not all values are associated with behaviours you might expect - for instance, 
-we do not make transcripts show up by tagging a file with "Transcript". See 
-[transcripts](../user-documentation/transcripts.md) instead.
+we do not make transcripts show up by tagging a file with "Transcript".
```
Comment (eldonquijote) [inline docs/concepts/node-media.md]: Islandora is not configured out-of-the box to display a transcript if you add a Media (file) and tag it as "Transcript".

File: docs/models/audio.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## Media and File Types
+
+Islandora Defaults uses the built-in Drupal "Audio" media type for video. Islandora Defaults configures it to accept files of type `mp3`, `wav`, and `aac`.
+
+## Derivatives
+
+Islandora uses Homarus (ffmpeg as a microservice) to create audio derivatives. Islandora Defaults sets you up to create:
+
+* "Service file" from the original file, with parameters `-codec:a libmp3lame -q:a 5`, stored in the public filesystem
```
Comment (eldonquijote) [inline docs/models/audio.md:11]: (these parameters can be changed in the configuration for the Drupal Action that Islandora uses to create an Audio derivative: ).

**Review by eldonquijote — COMMENTED** [review]
Thank you for these updates! I added some comments inline.

**Review by eldonquijote — APPROVED** [review]
LGTM

**Pull request merged**